website/content/docs/getting-started/deploy-and-login.mdx

```diff
@@ -0,0 +1,198 @@
+Terraform.
+
+Click **Deploy** after completing the form. It may take a few minutes to
+initialize the HCP Boundary instance.
+
+![Deploy boundary](/img/hcp-deploy-form.png)
+
+## Login to HCP Boundary
+
+Once the deployment is complete, the HCP Boundary instance can be accessed
+directly from the HCP Boundary portal using the credentials created during
+setup. 
+
+From the HCP Boundary portal page, click the **Open Admin UI** button. 
+
+Enter the admin username and password you created when you deployed the new
+instance and click **Authenticate**.
+
+The **Welcome to Boundary** setup wizard will launch automatically. There are
+two options for Boundary’s initial setup:
+
+1. Quick setup
+1. Manual
+
+![Setup Wizard](/img/setup-wizard.png)
+
+The **Quick setup** is recommended for new users, and generates a set of initial
+resources to assist in connecting to your first target. 
+
+![Setup Wizard](/img/setup-wizard-target.png)
+
+The **Manual** setup drops the user directly into Boundary, without any
+provisioned resources.
+
+Upon completion of the setup wizard you are logged into your HCP Boundary
+instance's Global scope via the web UI. This is the default scope for all new
+Boundary clusters.
+
+Click **Auth Methods**.
+
+Click the **copy icon** for the Password **Auth Method ID**. The ID is needed to
+authenticate to HCP Boundary using the CLI.
+
+<Tabs>
+<Tab heading="CLI">
+
+
+In the HCP Boundary portal, open the **Boundary Overview** page and click the
+**copy icon** in the **Copy this into Boundary Desktop** section. This copies
+your environment’s Boundary **Origin URL**.
+
+Open a terminal session and set the `BOUNDARY_ADDR` environment variable to the
+copied origin URL.
+
+```shell-session
+$ export BOUNDARY_ADDR=<boundary-origin-url>
+```
+
+Log in with the administrator credentials you created when you deployed the HCP
+Boundary instance.
+
+```shell-session
+$ boundary authenticate password \
+   -auth-method-id=COPIED_AUTH_ID \
+   -login-name=ADMIN_USERNAME \
+   -password=ADMIN_PASSWORD
+```
+
+You are now logged into your HCP Boundary instance's Global scope via the CLI.
+This is the default scope for all new Boundary clusters.
+
+If you are on Unix-like operating system (other than macOS/Darwin), you may get
+an error indicating that the token could not be stored, as the freedesktop.org
+Secret Service implementation is not always available. On these systems, you can
+work around this by installing `dbus-x11` and `gnome-keyring` using your package
+manager, then creating and unlocking the default keyring with the following,
+substituting in a password of your choice for "foobar" (but ending with `\n`).
+You can also avoid putting the password on the command line by running the
+`gnome-keyring-daemon` commands directly and entering in the password, followed
+by a newline (return) and an EOF (`Ctrl+D`).
+
+- `eval "$(printf 'foobar\n' | gnome-keyring-daemon --unlock)"`
```

Comment:
This is all quite complicated and much simpler to just use `pass`.

Reply:
This is the existing solution in the docs, would be happy to collaborate on an alternative. This was added due to the many comments in discuss regarding the keyring issue

Reply:
Unfortunately it's not really "the keyring issue" as every different keyring can have its own issue as they are different implementations. I'm not sure which discuss threads you're referring to but if they're relatively old they probably predate `pass` being made the default (or, possibly, even supported at all).

We should definitely not have this be the promoted solution. I didn't realize the docs had this in here at all. It also doesn't match the default behavior of the CLI which gives errors/warnings about `pass` if it doesn't work.

Reply:
what solution would you promote instead? Is documenting a solution for this issue even necessary with the implementation of `pass`?